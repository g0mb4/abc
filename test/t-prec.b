main() {
    extrn putchar, printn;
    auto a;

    /* 14 */
    a = 2 + 3 * 4;
    printn(a, 10);
    putchar('*n');

    /* 14 */
    a = 3 * 4 + 2;
    printn(a, 10);
    putchar('*n');

    /* 14 */
    a = (3 * 4) + 2;
    printn(a, 10);
    putchar('*n');

    /* 20 */
    a = (2 + 3) * 4;
    printn(a, 10);
    putchar('*n');
}
