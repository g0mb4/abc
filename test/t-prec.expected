14
14
14
20
